modules/standard/VisualDebug.chpl

```diff
@@ -53,25 +53,38 @@ module VisualDebug
 pragma "no doc"
   enum vis_op {v_start, v_stop, v_tag, v_pause};
 
-  private proc VDebugTree (what: vis_op, name: string, time: real, id: int = 0) {
-      var child = id * 2 + 1;
+private iter hc_id2com ( id: int, off: int ) {
```

Comment:
I think hc_id2com and VDebugTree could use a minor comment to describe what they do / what their purpose is. It took me a while to figure it out